model/schema/gene.yaml

```diff
@@ -86,3 +94,31 @@ slots:
     description: SOTerm describing gene type
     domain: Gene
     range: SOTerm
+
+  gene_designating_laboratory:
```

Comment:
I assume the intention of this slot is the same for Gene and GeneClass classes, so you can just use 'designating_laboratory' as defined in biologicalEntitySet.yaml and remove the `domain: GeneClass` line from the slot definition.  You can then either import biologicalEntitySet.yaml in gene.yaml or move the slot definition to core.yaml

Reply:
It's perfectly valid LinkML, but if the slots are intended to capture the same information then I don't see a need for duplication of code

Reply:
'fixed' or 'decided against' - I don't see any changes?